### ISSUE TITLE
[linux] Failed to watch for joystick connections in /dev/input

### ISSUE THREAD

**Issue by zugaldia** [mapbox/mapbox-gl-native#12223]
I'm trying to run the Linux test app on a Chromebook Pixel following the instructions on https://github.com/mapbox/mapbox-gl-native/blob/master/platform/linux/README.md from `master`. I'm using the Chrome OS dev channel that [experimentally supports Linux apps](https://developer.android.com/topic/arc/studio) within a Debian container.

Compilation works with no issues as far as I can tell but when I try `make run-glfw-app` I get the following error:

```
$ make run-glfw-app
platform/linux/ninja  -j4 -C build/linux-x86_64/Debug mbgl-glfw
ninja: Entering directory `build/linux-x86_64/Debug'
ninja: no work to do.
cd build/linux-x86_64/Debug && ./mbgl-glfw
[ERROR] {mbgl-glfw}[OpenGL]: GLFW error (65544): Linux: Failed to watch for joystick connections in /dev/input: No such file or directory
mbgl-glfw: /home/testdevices/mapbox-gl-native/platform/glfw/glfw_view.cpp:36: void glfwError(int, const char*): Assertion `false' failed.
Aborted
Makefile:364: recipe for target 'run-glfw-app' failed
make: *** [run-glfw-app] Error 134
```

/cc: @mapbox/embedded

**Comment by brunoabinader** [comment]
Ref: https://github.com/glfw/glfw/issues/833

This issue has been solved in https://github.com/glfw/glfw/commit/a397195d3fd14b3e94026bce684e6b5f392f5015 - however this change is scheduled to come in GLFW 3.3 version, which is not yet released. We can temporarily issue a hash-based version of GLFW in Mason to solve that.

**Comment by zugaldia** [comment]
@brunoabinader thanks for the quick eyes and glad to hear a fix is coming - we can wait until 3.3 gets released, at least on my end this isn't urgent.
